### ISSUE TITLE
Refine “Followed content scope” by contact

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#12127]
So far I had been using the widest followed content scope possible and it was manageable. However, with the recent Twitter migration, one of my contacts who’s also an instance admin has been greeting the new accounts on his instance more than usual.

I’d like to reduce the followed content scope but just for him for a while.

**Comment by tobiasd** [comment]
You mean a user group, but instead of "adding sources into the group" you want to "exclude sources"? Sounds nice.

**Comment by MrPetovan** [comment]
No, rather that I want to get only top-level posts from this contact for a while, but keep the default for all my other contacts.

**Comment by tobiasd** [comment]
Ah, ok. So "Settings -> Social Networks -> General Social Media Settings -> Followed content scope" on a per contact level.

**Comment by MrPetovan** [comment]
Exactly.

**Comment by ne20002** [comment]
I really would like this. I also have one contact where I'm interesstend in things she post but she is also boosting whatever she sees an having only this disabled for this one contact would be great. :)

I wonder how exactly the interface/communication between friendica and a kind a generic ActivityPub app like Tusky is working.
I see that I can filter certain things in Tusky. But I believe that the data send to Tusky by a friendica server is also already filtered on the server by the settings I do on my friendica account. Am I right?
So having additional filtering options would be great. With the Twexit happening I see much more messages than before ... which leads to the need to filter more to focus on what is really relevant.
